packages/sdk/src/actions/callOnExtension.test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "vitest";
+import { encodeAbiParameters, encodeFunctionData, getAddress, parseAbiParameters, toHex } from "viem";
+import { setupAnvil } from "../../tests/anvil.js";
+import { decodeCallOnExtensionParams, prepareCallOnExtensionParams } from "./callOnExtension.js";
+import { AAVE_V2_ADAPTER, ALICE, A_WETH, BOB, INTEGRATION_MANAGER, WETH } from "../../tests/constants.js";
+import { testActions, sendTestTransaction } from "../../tests/globals.js";
+import { toWei } from "../utils/conversion.js";
+import { IntegrationManagerActionId } from "../enums.js";
+import { lendSelector } from "../../tests/selectors.js";
+
+setupAnvil();
+
+test("call on extension should work correctly", async () => {
+  const vaultOwner = ALICE;
+  const sharesBuyer = BOB;
+
+  const { comptrollerProxy, vaultProxy } = await testActions.createTestVault({
+    vaultOwner,
+    denominationAsset: WETH,
+  });
+
+  const depositAmount = toWei(250);
+
+  await testActions.buyShares({
+    comptrollerProxy,
+    sharesBuyer,
+    investmentAmount: depositAmount,
+  });
+
+  const integrationData = encodeAbiParameters(parseAbiParameters("address aToken, uint depositAmount"), [
+    A_WETH,
+    depositAmount,
+  ]);
+
+  const callArgs = encodeAbiParameters(parseAbiParameters("address adapter, bytes4 selector, bytes integrationData"), [
+    AAVE_V2_ADAPTER,
+    lendSelector,
+    integrationData,
+  ]);
```

Comment:
Can we also get utils for this? So end users wouldn't have to write this themselves?